worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/ScriptEngineType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016-2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.worker.document.model;
+
+public enum ScriptEngineType
```

Comment:
Interfaces, methods, parameters, etc. in the interface module should have comprehensive JavaDoc since they're used by all the workers (i.e. not just internal code).